src/audio/google_rtc_audio_processing.c

```diff
@@ -34,8 +34,9 @@
 
 #include <google_rtc_audio_processing.h>
 #include <google_rtc_audio_processing_platform.h>
+#include <google_rtc_audio_processing_sof_message_reader.h>
 
-#define GOOGLE_RTC_AUDIO_PROCESSING_SAMPLERATE 48000
+#define GOOGLE_RTC_AUDIO_PROCESSING_FRAMES_PER_SECOND 100
```

Comment:
I think this name is false, based on the usage it looks like its better to make this macro called FREQENCY_TO_PERIOD_FRAMES

Reply:
Sounds good.
Done.